system/settings/settings.xml

```diff
@@ -661,6 +661,16 @@
           <default>0</default>
           <control type="slider" format="percentage" range="0,100" />
         </setting>
+        <setting id="subtitles.linespacing" type="integer" label="39202">
+          <level>3</level>
+          <default>0</default>
+          <constraints>
+            <minimum>0</minimum>
+            <step>1</step>
+            <maximum>40</maximum>
+          </constraints>
+          <control type="spinner" format="integer"/>
+        </setting>
```

Comment:
like the other settings, this also needs to be done as a percentage with 0-100 range
this is because we don't want to have settings that depends to a specific library